starter-code/scripts/models/article.js

```diff
@@ -21,7 +21,7 @@
   // Set up a DB table for articles.
   Article.createTable = function() {
     webDB.execute(
-      '', // TODO: What SQL command do we run here inside these quotes?
+      'CREATE TABLE articles(title, category, author, authorUrl, publishedOn, body);', // DONE: What SQL command do we run here inside these quotes?
```

Comment:
You didn't provide any data types for your input columns. You should go ahead and do that.